docs/clients.md

```diff
@@ -1,83 +1,70 @@
-# Ultra Client Side SDK
+# Ultra Client Side SDK documentation
+Here are the details of the Client side APIs you need to incorporate while integration of your app with Ultra.
 
-> Latest JS SDK version : [1.0.0-beta](https://www.npmjs.com/package/fk-platform-sdk)
+> Use this latest version of JS SDK: [1.0.0-beta](https://www.npmjs.com/package/fk-platform-sdk)
 
-## Overview
-
-
-This SDK enables developers to build applications that run inside Flipkart app.
-
-All the methods mentioned here will work with both React Native and Webview. 
-All the methods are asynchronous in nature and will always return a promise that gets resolved with the values. Fire and forget calls are an exception where you may not care about the response.
-
-## Getting Started
+## Integrate JS SDK
 #### Add the dependency
-If using node, add this repository as an npm package (Both webview and react-native)
+If you are using a node, add the following repository as an NPM package for both WebView and React Native platforms. Visit [here](https://www.npmjs.com/package/fk-platform-sdk) to get the latest SDK for this repository.
 
 ```
 npm install --save fk-platform-sdk
 ```
 
-You can also visit [the NPM](https://www.npmjs.com/package/fk-platform-sdk) page for the SDK 
-
-Alternatively, if you are using webview only you can also include the following script directly inside a `<script>` tag:
+Or if you are using only a web browser without node, include the following script directly inside the `<script>` tag:
```

Comment:
Please replace "only a web browser without node" with "webview only"

Reply:
Done, please check the latest commit.